packages/genesys-spark-components/src/components/beta/gux-avatar/tests/gux-avatar.spec.ts

```diff
@@ -61,4 +61,17 @@ describe('gux-avatar', () => {
       }
     );
   });
+
+  describe('#render different accents', () => {
+    ['default', 'auto', 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12].forEach(
+      (accent: string) => {
+        it(`should work as expected for "${accent}"`, async () => {
+          const html = `<gux-avatar-beta name="John Doe" accent=${accent}></gux-avatar-beta>`;
+          const page = await newSpecPage({ components: [GuxAvatar], html });
+
+          expect(page.root).toMatchSnapshot();
+        });
+      }
+    );
+  });
 });
```

Comment:
I know it feels a bit redundant but you should also create an e2e test file

Reply:
done